__tests__/app.test.js

```diff
@@ -203,3 +203,63 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH: 201 when receiving a positive integer it increments  the votes on the requested article and returns the article with updates votes ", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .send({ inc_votes: 2 })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.article.votes).toBe(2);
+        expect(body.article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
We can be more specific with our assertion against the values here, we know what all the values should look like after the patch so lets add them all in and then we can be sure the patch has only affected the votes property 👍